kod/object/passive/spell/creaweap.kod

```diff
@@ -108,27 +192,25 @@ messages:
                }
             }
          }
-      }
+      }        
 
       % Weapon has half its hits, + iSpellPower% of the other half.
       Send(oWeapon,@SetHits,#number=(Send(oWeapon,@GetHits)*(iSpellPower+100))/200);
 
       if Send(who,@ReqNewHold,#what=oWeapon)
       {
-	      Send(who,@NewHold,#what=oWeapon);
-	      Send(who,@MsgSendUser,#message_rsc=createweapon_cast_rsc,
-	           #parm1=send(oWeapon,@GetCapIndef),#parm2=send(oWeapon,@GetName));
+         Send(who,@NewHold,#what=oWeapon);
+         Send(who,@MsgSendUser,#message_rsc=createweapon_cast_rsc,
+         #parm1=send(oWeapon,@GetCapIndef),#parm2=send(oWeapon,@GetName));
       }
       else
       {
-	      Send(oWeapon,@Delete);
-	      Send(who, @MsgSendUser, #message_rsc=createweapon_inv_full_rsc,#parm1=send(oWeapon,@GetCapIndef),#parm2=send(oWeapon,@GetName));
+         Send(oWeapon,@Delete);
+         Send(who, @MsgSendUser, #message_rsc=createweapon_inv_full_rsc,#parm1=send(oWeapon,@GetCapIndef),#parm2=send(oWeapon,@GetName));
       }
       
       oItemAtt = send(SYS,@FindItemAttByNum,#Num=IA_MADE);
```

Comment:
This version of create weapon still has a bug where the item attribute is applied to a deleted weapon, which causes a lot of timer errors when it expires and on system saves. Can either apply the itematt beforehand or (what we did on 103) have the weapon drop onto the floor instead of the player cannot hold it.